docs/data/components/tooltip/UnstyledTooltipFollowCursor.js

```diff
@@ -7,7 +7,7 @@ import { styled } from '@mui/system';
 export default function UnstyledTooltipFollowCursor() {
   return (
     <div style={{ display: 'flex', gap: 12 }}>
-      <Tooltip.Root followCursorAxis="both">
+      <Tooltip.Root trackCursorAxis="both">
```

Comment:
It's not clear to me why it started to fail now, but I guess it's a good thing it fails.